src/plugins/multimedia/_base.scss

```diff
@@ -182,6 +187,15 @@
 		text-align: right;
 	}
 
+	summary {
```

Comment:
Suggested changes:

```
summary, details[open] { }
```

Reply:
That's not in line with how WET has traditionally dealt with this kind of situation. Selector rules are usually placed on their own lines and separated as much as possible. That's why I had resorted to ``&[open]`` and a placeholder selector instead of ``details[open]``).

In any case, I'll go with the way you're suggesting to simplify the code and since I don't feel super strongly about it.